packages/text-annotator-react/src/TextAnnotator.tsx

```diff
@@ -7,26 +8,27 @@ import '@recogito/text-annotator/dist/text-annotator.css';
 
 export type TextAnnotatorProps<E extends unknown> = TextAnnotatorOptions<E> & {
 
-  children?: ReactNode | JSX.Element; 
+  children?: ReactNode | JSX.Element;
 
   filter?: Filter;
 
   style?: DrawingStyle | ((annotation: TextAnnotation) => DrawingStyle);
 
 }
 
-export const TextAnnotator = <E extends unknown>(props: TextAnnotatorProps<E>) => {
+export const TextAnnotator = forwardRef(<E extends unknown>(props: TextAnnotatorProps<E>, ref: ForwardedRef<HTMLDivElement>) => {
```

Comment:
I didn't use the generic type of the `forwardRef`, because it would allow using the `<E extends unknown>` for the `TextAnnotatorProps`